### DISCUSSION TITLE
I translated Thai for Thai people. I'd like to ask you to consider and add it to the cmi project.

### DISCUSSION THREAD

**Discussion by var1nthon** [Zrips/CMI#8975] [Show and tell]
Translations:
[Locale_TH.txt](https://github.com/user-attachments/files/16357037/Locale_TH.txt)

DeathMessages:
[Locale_TH.txt](https://github.com/user-attachments/files/16357108/Locale_TH.txt)

**Comment by CloudeLecaw** [comment]
<3

**Comment by CloudeLecaw** [comment]
might the wrong hithub..but you also made a TH Translation for CMILIB?
just asking because you know, all zrips plugins use cmilib and also the parts of its translations.
